common/src/main/java/com/vmware/connectors/common/web/SingleDeferredResult.java

```diff
@@ -3,8 +3,9 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-package com.vmware.connectors.common.utils;
+package com.vmware.connectors.common.web;
 
+import com.vmware.connectors.common.web.DeferredResultSubscriber;
```

Comment:
Unnecessary import?

Reply:
I guess PMD needs fixing in this project as well.

Reply:
It turns out that pmd isn't running as part of the build.  It's due to something really silly, however, after turning it on, I'm finding lots of places that need code cleanup, so I won't do it in this PR.

I'll merge this and create a ticket to fix the issues.